Use `easy_install` to install `pip` on all platforms

MattiSG opened the pull request:
@fpagnoux If I remember well, you had to `apt-get` `pip` on a clean Ubuntu, right? And @noirbizarre recommended to install it with a Python tool, hence this patch.

Please confirm this would do the trick  :)

MattiSG committed:
Use easy_install to install pip on all platforms

fpagnoux commented:
I confirm, I used `easy_install pip` on ubuntu, which downloads the most recent pip version, (`apt-get` was downloading 1.0).
Careful though, on a clean machine `apt-get install python-setuptools` is a prerequisite to use easy_install, I will add it to ubuntu bootstrap

MattiSG commented:
> I will add it to ubuntu bootstrap

:+1:

You can add this straight to this PR by checking out this branch and pushing to it  :)

Commit:
Fix ubuntu bootstrap script

- Install the packages without user interaction (--assume-yes)
- Don't install pip with apt get, but with easy_install in the common bootrstrap script
- Update apt-get before installing anything
